xplique/example_based/projections/base.py

```diff
@@ -0,0 +1,254 @@
+"""
+Base projection for similar examples in example based module
+"""
+
+from abc import ABC
+import warnings
+
+import tensorflow as tf
+import numpy as np
+
+from ...commons import sanitize_inputs_targets, get_device
+from ...types import Callable, Union, Optional
+
+
+class Projection(ABC):
```

Comment:
We can remove the `abc` package here as it does not define any abstractmethods it should not inherits from ABC